Sources/Operators/AssignWeakly.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  AssignWeakly.swift
```

Comment:
No sure if this should be called `AssignWeakly`. It's not just about the weakly part.

Reply:
I've renamed it to just `Assign`